duckchat/duckchat-impl/src/main/res/layout/activity_duck_chat_settings.xml

```diff
@@ -105,9 +105,10 @@
 
                     <ImageView
                         android:id="@+id/duckAiInputScreenToggleWithoutAiImage"
-                        android:layout_width="match_parent"
+                        android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
-                        android:scaleType="fitCenter"
+                        android:scaleType="fitXY"
```

Comment:
Had to change this because the ImageView container would be slightly taller than the image itself and the touch ripple would go outside the bounds.